.github/workflows/deployment.yml

```diff
@@ -54,34 +54,6 @@ jobs:
       - name: Timezone 🌐
         run: echo "Timezone set to ${{ env.TIMEZONE }}"
 
-  database:
-    name: Database 📦️
-    needs: setup
-    environment: ${{ needs.setup.outputs.environment }}
-    env:
-      NAME: mysql
-      ENVIRONMENT: ${{ needs.setup.outputs.environment }}
-    runs-on: [self-hosted, EXIP, deployment]
-
-    steps:
-      - name: Repository 🗃️
-        uses: actions/checkout@v4
-
-      - name: Azure 🔐
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-
-      - name: Defaults ✨
-        uses: Azure/cli@v1.0.8
-        with:
-          inlineScript: |
-            # Basic
-            az configure --defaults location=${{ vars.REGION }}
-            az configure --defaults group=rg-${{ env.PRODUCT }}-${{ github.ref_name }}-${{ vars.VERSION }}
-
-    #TODO: EMS-1176: MS SQL DB setup
-
```

Comment:
Is this removal intentional?

Reply:
Yes, we will be deploying straight to AMI.